### ISSUE TITLE
debug_lines false positive for string.printable

### ISSUE THREAD

**Issue by zupo** [plone/plone.recipe.codeanalysis#95]
A line like this one triggers a debug_lines warning because it contains the `print` string, but it's valid code and should not trigger a warning:

``` python
text = filter(lambda x: x in string.printable, text)
```
